[2021-12-08 11:20:59,240] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-04T09:00:00+00:00 [queued]>
[2021-12-08 11:20:59,248] {taskinstance.py:670} INFO - Dependencies all met for <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-04T09:00:00+00:00 [queued]>
[2021-12-08 11:20:59,248] {taskinstance.py:880} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 11:20:59,248] {taskinstance.py:881} INFO - Starting attempt 1 of 1
[2021-12-08 11:20:59,248] {taskinstance.py:882} INFO - 
--------------------------------------------------------------------------------
[2021-12-08 11:20:59,255] {taskinstance.py:901} INFO - Executing <Task(TwitterOperator): twitter_aluraonline> on 2021-12-04T09:00:00+00:00
[2021-12-08 11:20:59,256] {standard_task_runner.py:54} INFO - Started process 22070 to run task
[2021-12-08 11:20:59,307] {standard_task_runner.py:77} INFO - Running: ['airflow', 'run', 'twitter_dag', 'twitter_aluraonline', '2021-12-04T09:00:00+00:00', '--job_id', '12', '--pool', 'default_pool', '--raw', '-sd', 'DAGS_FOLDER/twitter_dag.py', '--cfg_path', '/tmp/tmpp6gr_m8h']
[2021-12-08 11:20:59,307] {standard_task_runner.py:78} INFO - Job 12: Subtask twitter_aluraonline
[2021-12-08 11:20:59,325] {logging_mixin.py:112} INFO - Running <TaskInstance: twitter_dag.twitter_aluraonline 2021-12-04T09:00:00+00:00 [running]> on host rafael-Aspire-A515-54G
[2021-12-08 11:20:59,364] {base_hook.py:89} INFO - Using connection to: id: twitter_default. Host: https://api.twitter.com, Port: None, Schema: None, Login: None, Password: None, extra: XXXXXXXX
[2021-12-08 11:20:59,365] {twitter_hook.py:36} INFO - URL: https://api.twitter.com/2/tweets/search/recent?query=AluraOnline&tweet.fields=author_id,conversation_id,created_at,id,in_reply_to_user_id,public_metrics,text&expansions=author_id&user.fields=id,name,username,created_at&start_time=2021-12-04T09:00:00.00Z&end_time=2021-12-05T09:00:00.00Z
[2021-12-08 11:20:59,648] {logging_mixin.py:112} WARNING - /home/rafael/Documentos/alura/datapipeline/.env/lib/python3.7/site-packages/urllib3/connectionpool.py:988: InsecureRequestWarning: Unverified HTTPS request is being made to host 'api.twitter.com'. Adding certificate verification is strongly advised. See: https://urllib3.readthedocs.io/en/latest/advanced-usage.html#ssl-warnings
  InsecureRequestWarning,
[2021-12-08 11:21:00,636] {taskinstance.py:1070} INFO - Marking task as SUCCESS.dag_id=twitter_dag, task_id=twitter_aluraonline, execution_date=20211204T090000, start_date=20211208T142059, end_date=20211208T142100
[2021-12-08 11:21:04,249] {local_task_job.py:102} INFO - Task exited with return code 0
